core/camera/src/main/java/com/google/jetpackcamera/core/camera/CameraXCameraUseCase.kt

```diff
@@ -118,6 +118,8 @@ constructor(
 
     private val _surfaceRequest = MutableStateFlow<SurfaceRequest?>(null)
 
+    private val cameraSelector = MutableStateFlow<CameraSelector?>(null)
```

Comment:
This can be a local variable inside the coroutine. Generally we want to try to avoid varaibles with scope longer than the coroutine if possible.

Reply:
i'd still need to be aware of changes to the lensfacing setting within the CameraSession. My first thought was to place the `CameraAppSettings.cameraLensFacing` in the Transient settings, but I recall you advising against including settings that would rebind the camera. What do you think would be a better location to place it?